extensions/amp-connatix-player/0.1/amp-connatix-player.js

```diff
@@ -71,10 +105,85 @@ export class AmpConnatixPlayer extends AMP.BaseElement {
         // Ignore messages from other iframes.
         return;
       }
-      // Player wants to close because the user interracted on its close button
-      if (getData(e) === 'cnx_close') {
-        this.destroyPlayerFrame_();
-        this.attemptCollapse();
+      const dataString = getData(e);
+      const dataJSON = tryParseJson(dataString) || dict();
```

Comment:
Nit: If the `tryParseJson()` fails, this returns an empty `dict`.

Would this also be a early return candidate?